ui/selectors/transactions.js

```diff
@@ -21,6 +22,20 @@ const INVALID_INITIAL_TRANSACTION_TYPES = [
   TransactionType.retry,
 ];
 
+// The statuses listed below are permitted in the Activity list for Smart Swaps because
+// a Smart Swap with these statuses hasn't been added to the TransactionController's regular transaction list yet.
+// The SUCCESS status is not allowed because once a Smart Swap is successful,
+// the "this.confirmExternalTransaction" function is called, which moves the transaction
+// to the regular transaction list in the TransactionController.
```

Comment:
Hey, I believe we can simplify this a bit to make it clearer. What are your thoughts?
```suggestion
// The statuses listed below are allowed in the Activity list for Smart Swaps
// The SUCCESS and REVERTED statuses are excluded because smart transactions with 
// those statuses are already in the regular transaction list.
```